modules/shaded-libs/wss4j-ws-security-dom/pom.xml

```diff
@@ -50,22 +51,14 @@
                                 </relocation>
                             </relocations>
                             <artifactSet>
-                                <excludes>
-                                    <exclude>com.google.code.findbugs:jsr305:jar:</exclude>
-                                    <exclude>com.google.guava:guava:jar:</exclude>
-                                    <exclude>commons-codec:commons-codec:jar:</exclude>
-                                    <exclude>joda-time:joda-time:jar:</exclude>
-                                    <exclude>net.shibboleth.utilities:java-support:jar:</exclude>
-                                    <exclude>org.apache.httpcomponents:httpclient:jar:</exclude>
-                                    <exclude>org.apache.httpcomponents:httpcore:jar:</exclude>
-                                    <exclude>org.opensaml:opensaml-core:jar:</exclude>
-                                    <exclude>org.slf4j:slf4j-api:jar:</exclude>
-                                </excludes>
+                                <includes>
+                                    <include>org.apache.wss4j:wss4j-ws-security-dom</include>
```

Comment:
Because we only need to have this jar inside the shaded lib, so instead of excluding all the other jars, we can only include that.